irods/test/user_group_test.py

```diff
@@ -552,10 +552,41 @@ def alice_login(pw):
             if user_alice:
                 user_alice.remove()
 
+    def test_set_and_query_group_quota(self):
+        #import pdb
+        #pdb.set_trace()
+        bob = quota_group = None
+        ses = self.sess
+        test_object = None
+        my_quota = 10000
+        my_object_size = 1400
+        try:
+            bob = self.sess.users.create('bob', 'rodsuser')
+            ses.users.modify('bob', 'password', 'bpass')
+            quota_group = self.sess.groups.create('quota_group_for_bob')
+            quota_group.addmember(bob.name)
+            quota_group.set_quota(my_quota)
+            with iRODSSession(user = 'bob', password = 'bpass', host = ses.host, port = ses.port, zone = ses.zone) as user_sess:
+               test_object = user_sess.data_objects.create('/tempZone/home/public/bob_file_testing_group_quota')
+               with test_object.open('w') as f:
+                   f.write(b'_' * my_object_size)
+            ses.groups.calculate_usage()
```

Comment:
Got some questions about `*.groups.calculate_usage()`.

Q. How does this work when there are multiple groups?
Q. Does it update the quotas for **all** groups in the zone or session?
Q. How does it know which groups to update quotas for?

Reply:
It's the equivalent for  iadmin cu ... doesn't that just do a general calculation against all existing quotas?

Reply:
You're right. Carry on.